ops/Dockerfile

```diff
@@ -2,11 +2,14 @@ FROM public.ecr.aws/docker/library/node:18.16-alpine
 
 WORKDIR /cdk
 
-COPY package.json yarn.lock ./
+COPY package.json pnpm-lock.yaml ./
 
-RUN yarn install --frozen-lockfile
+# pnpm isn't installed by default
+# https://pnpm.io/installation#using-corepack
+RUN corepack enable
```

Comment:
The only bummer with this is that the `corepack enable` layer will be invalidated in the cache by an update to your package.json that's unrelated to the pnpm version. This means a slow-down for iterative builds, or less caching in your image store.

This is a trade-off and may not be an issue in your context.